src/app/controllers/sap.php

```diff
@@ -53,7 +53,7 @@ public function login() {
             if (isset($_POST['csrf_token'])) {
                 $recvd_token = $_POST['csrf_token'];
             } else {
-                $recvd_token = NULL;
+                $recvd_token = null;
```

Comment:
No reason. Not much for or against it I guess. And no point in this commit anyway. :P